ethereum/spec/src/main/java/tech/pegasys/teku/spec/schemas/SchemaDefinitions.java

```diff
@@ -66,11 +73,24 @@ default AttestationSchema getAttestationSchema() {
 
   AttesterSlashingSchema getAttesterSlashingSchema();
 
+  @NonSchema
+  Optional<SignedBeaconBlockUnblinder> createSignedBeaconBlockUnblinder(
+      SignedBeaconBlock signedBlindedBeaconBlock);
+
+  @NonSchema
+  Optional<SignedBeaconBlockBlinder> getSignedBeaconBlockBlinder();
```

Comment:
hmm, I'm not entirely convinced these belong in `SchemaDefinitions`. I can see the reasoning for it because they make a lot of use of schema definitions, but it feels like we've expanded the scope of `SchemaDefintiions` from providing just the definitions to providing logic now as well and that should really come from something like `BlockProposalUtil`.

Reply:
🤔 right. Placing them in `BlockProposalUtil` seems more natural.

Reply:
not really:) I'll found a better place